src/client/client.c

```diff
@@ -1748,6 +1748,28 @@ static void Usage(void)
 #endif
 }
 
+#ifndef USE_WINDOWS_API
+int checkStdin(void)
+{
+    int stop = 0;
+    struct timeval timeout;
+    timeout.tv_sec = 0;
+    timeout.tv_usec = 0;
+ 
+    fd_set readfds;
```

Comment:
variable not declared at the top of the scope

Reply:
Moved